planq/assetlist.json

```diff
@@ -29,6 +29,32 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/planq/images/planq.svg"
         }
       ]
+    },
+    {
+      "description": "The token of Source Protocol bridged from BSC.",
```

Comment:
Since this is bridged, can we added the traces[] point back to it origin? I imagine it might be something like Source -> BSC > Planq?

Reply:
I'm a bit unsure what to do here. It's an ERC20 token originating from BSC. This got then bridged to Planq.
So for now its BSC > Planq.

The traces def will fall out of schema since its not originating from ibc. This is what I can add though:

"traces": [
        {
          "type": "erc20",
          "counterparty": {
            "chain_name": "bsc",
            "base_denom": "erc20/srcx",
            "channel_id": "0"
          },
          "chain": {
            "channel_id": "",
            "path": ""
          }
        }
      ],

Let me know, thanks

Reply:
This is not quite right. the type: would be `bridge`; there is a $def for non-ibc transfers. It take bridge provider and contract addresses.
The srcx token will need to be defined in /non-cosmos/binancesmartchain/assetlist.json.
Just to make sure, the Source Protocol here is native to BSC, and it's not bridged from some other network to get to BSC?

Reply:
>Just to make sure, the Source Protocol here is native to BSC, and it's not bridged from some other network to get to BSC?
Correct.

Made adjustments like you suggested.